com.obsproject.Studio.Plugin.RewardsTheater.yaml

```
@@ -0,0 +1,58 @@
+id: com.obsproject.Studio.Plugin.RewardsTheater
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.5
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/RewardsTheater
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - '*.a'
+modules:
+  - name: boost
+    buildsystem: simple
+    build-commands:
+    - ./bootstrap.sh --prefix=/app/plugins/RewardsTheater --with-libraries=system,url,json
+    - ./b2 -j $FLATPAK_BUILDER_N_JOBS install
+    cleanup: "*"
```

```suggestion
    cleanup: 
      - "*"
```

Sorry, I typoed the cleanup.

---

I just realized the cleanup like you suggested it removes all the .so files generated by Boost, so probably it's not the best idea to include it at all?

---

Hm, looks like it links against libboost_url and libboost_json. I was under the impression it's a build time dependency as boost usually is, but looks like that's not the case.

---

there should be no .so because you should be building boost for static linking. Then the cleanup should get rid of the .a

---

No, boost is shared libs by default, fmt is being built as static

---

I said:
>  you should be building boost for static linking. 

And that's not a suggestion.

---

Ah sorry, I misread. Any reason for that? shared is the upstream default.

---

it's an extension. there is no path and it shouldn't try to load a different one. This is a general rule.

---

> it's an extension. there is no path and it shouldn't try to load a different one. This is a general rule.

Hm, makes sense. I can see it conflicting with other versions of boost.

---

Fixed, now it links to Boost statically